qermit/zero_noise_extrapolation/zne.py

```diff
@@ -588,7 +585,9 @@ def task(
 
             # This compilation pass was added to account for the case that
             # the inverse of a gate is not in the gateset of the backend.
+            rebase_pass = backend.rebase_pass()
             rebase_pass.apply(zne_circ)
```

Comment:
not worth redoing the code for really, but why not just `backend.rebase_pass().apply(zne_circ)`?